### PR TITLE
Update EddieHub Discord link in README

### PR DESCRIPTION
**Pull request by Evavic44**
closes #ISSUE-NO 

#### What does this PR do?
Updates Eddiehub Discord link in README
```diff
- https://discord.gg/jZQs6Wu
+ (http://discord.eddiehub.org/
```
#### How can this be manually tested?
By clicking the link in the README

#### Any background context you want to provide?
Eddiehub discord link was changed on two different lines in the README file. 
#### What is the relevant issue link?
#1401

### CODE REVIEW

**Review by schmelto — DISMISSED** [review]

**Comment by schmelto** [comment]
Thanks for the PR looking good for me 👍🏼

**Comment by Evavic44** [comment]
> Thanks for the PR looking good for me 👍🏼

Thank you too.

File: .github/ISSUE_TEMPLATE/gh-profile-review.md [hunk 1]
```diff
@@ -9,6 +9,6 @@ assignees: ''
 **Github Profile Link:**
 <!-- your github profile link goes here. -->
 
-You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)
+You can also join the discord community [here](http://discord.eddiehub.org)
```
Comment (kaiwalyakoparkar) [inline .github/ISSUE_TEMPLATE/gh-profile-review.md:12]: I think it should be https://discord.eddiehub.org

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
Please change the link to all the files

File: .github/ISSUE_TEMPLATE/gh-profile-review.md (hunk 1 above)
Comment (schmelto) [reply .github/ISSUE_TEMPLATE/gh-profile-review.md:12]: No it's `http:` so those changes are good :) see [here](https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/issues/163#issuecomment-826338069)

Comment (kaiwalyakoparkar) [reply .github/ISSUE_TEMPLATE/gh-profile-review.md:12]: Oh, thanks for letting me know. Actually, when I pasted it in the browser it automatically turned into `https:` so I thought it was to be `https:`

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM

**Review by schmelto — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Thank you all 👍

**Pull request merged**